iib/workers/tasks/fbc_utils.py

```diff
@@ -17,6 +19,45 @@ def is_image_fbc(image):
     :return: True if image is FBC type, False otherwise (SQLite)
     :rtype: bool
     """
+    from iib.workers.tasks.utils import skopeo_inspect
+
     skopeo_output = skopeo_inspect(f'docker://{image}')
     dc_image_label = 'operators.operatorframework.io.index.configs.v1'
     return dc_image_label in skopeo_output['Labels']
+
+
+def _get_index_fbc_dir(from_index, base_dir):
+    """
+    Get file-based catalog directory from the specified index image and save it locally.
+
+    :param str from_index: index image to get database file from.
+    :param str base_dir: base directory to which the database file should be saved.
```

Comment:
I am a little confused here, do we get the database file from the FBC index image? Can you probably elaborate what you mean by "database file" in these 2 lines?

Reply:
copy-paste error ... I forgot to rewrite it to FBC

Reply:
Are we going to have this function or it would be removed?